stdlib/messaging/rabbitmq/src/main/ballerina/rabbitmq/channel.bal

```diff
@@ -71,8 +71,8 @@ public type Channel client object {
     # + message - The message body.
     # + exchange - The name of the exchange the message is published to.
     # + return - Returns an error if an I/O error is encountered or nil if successful.
-    public remote function basicPublish(string message, string routingKey,
-                                            string exchange = "") returns error? = external;
+    public remote function basicPublish(int | float | byte | boolean | string | map<any> | xml | json | byte[] message,
```

Comment:
Shall we define a new type, a union of all these types and use it here? So that we reuse it if needed
```bal
type Message int | float | byte | boolean | string | map<any> | xml | json | byte[];
```

Reply:
Fixed